internal/pkg/service/stream/storage/node/coordinator/clusterlock/file.go

```diff
@@ -3,21 +3,26 @@ package clusterlock
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/log"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/distlock"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop"
 	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
 )
 
 func LockFile(ctx context.Context, locks *distlock.Provider, logger log.Logger, fileKey model.FileKey) (lock *etcdop.Mutex, unlock func(), err error) {
 	lock = locks.NewMutex(fmt.Sprintf("operator.file.%s", fileKey))
 	if err := lock.Lock(ctx); err != nil {
-		logger.Errorf(ctx, "cannot lock %q: %s", lock.Key(), err)
-		return nil, nil, err
+		return nil, nil, errors.PrefixErrorf(err, "cannot lock %q:", lock.Key())
 	}
 
+	logger.Debug(ctx, "acquired lock")
```

Comment:
- `lock.key` should by an attribute (also in the cannot lock)
- for now, it should be `info` (it will be a maximum of hundreds of messages per second).
- there should be also `released lock` message.